tmpl/wiki/faq.md

```diff
@@ -1,25 +1,24 @@
 
+We are also conducting research into novel databases, filesystems and security technologies at Cambridge that use MirageOS as the baseline to build complex distributed systems.
 
 *Q: How does MirageOS compare against other cloud-friendly OS options (e.g like OSv) and also different approaches like like containers (e.g Docker)?*
 
 MirageOS represents our desire for a radically simpler way of building complex distributed systems using a modern modular, functional and type-safe programming language such as OCaml. Unlike other cloud-friendly operating systems such as OSv, we do not attempt to optimize *existing* code, but instead focus on a toolkit to make it easier to quickly assemble *new* systems without having to be a domain expert in (e.g.) kernel programming.
 
-The downside to our approach is that we only work with open protocols, since we cannot build clean-slate versions of closed protocols for which we have no specification. On the other hand, the 1.0 release contains clean-slate libraries for TCP/IP, DNS, Xen device drivers, VNC, HTTP and other common Internet protocols, but all written in a completely type-safe fashion so that they are resistant to attacks such as buffer overflows that are plaguing the Internet. There's a good chance that a few years from now, existing systems will still be suffering those attacks, but MirageOS will continue to grow and mature its protocol implementations without sacrificing safety.
+The downside to our approach is that we only work with open protocols, since we cannot build clean-slate versions of closed protocols for which we have no specification.  Instead, we hope to build a system which enables easy development of arbitrary protocols and integration of them into the stack; a measure of our success is whether users can implement such protocols themselves, without specialised systems or kernel knowledge.
 
+On the other hand, the 2.9 release contains clean-slate libraries for TLS, TCP/IP, DNS, Xen device drivers, VNC, HTTP and other common Internet protocols, but all written in a completely type-safe fashion so that they are resistant to attacks such as buffer overflows that are plaguing the Internet. There's a good chance that a few years from now, existing systems will still be suffering those attacks, but MirageOS will continue to grow and mature its protocol implementations without sacrificing safety.
```

Comment:
is the `2.9` here relevant? it will likely be out of date within weeks ;) maybe use `latest` or `current` instead?